chrome-extension/src/eventPage.ts

```diff
@@ -9,3 +9,29 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
 
   return isResponseAsync;
 });
+
+chrome.contextMenus.create({
+  title: "Save Image",
+  contexts: ["image"],
+  onclick: addPlace,
+});
+
+function addPlace(image: chrome.contextMenus.OnClickData) {
+  const imageUrl = image.srcUrl;
+  console.log("clicked image");
+  chrome.storage.sync.set({imageUrl: imageUrl}, function() {
+    console.log('Value is set to ' + imageUrl);
+    var w = 700;
+    var h = 700;
+    var left = (screen.width/2)-(w/2);
```

Comment:
Does this mean the size for every image is the same? Or would the screen size change based on the image size itself?